src/test/java/teammates/test/cases/action/AdminEmailLogPageActionTest.java

```diff
@@ -81,8 +80,12 @@ private void loadLogMessages() {
     }
 
     private void initVariable() {
-        formatterAdminTime = new SimpleDateFormat("dd/MM/yy");
-        formatterAdminTime.setTimeZone(TimeZone.getTimeZone(Const.SystemParams.ADMIN_TIME_ZONE));
+        formatterAdminTime = DateTimeFormatter.ofPattern("dd/MM/yyyy");
```

Comment:
Why not just make `formatterAdminTime` static and set it at initialization since `DateTimeFormatter` objects are immutable? Then we can remove `initVariable`.